test/controllers/integrations/buildkite_controller_test.rb

```diff
@@ -63,4 +63,29 @@
       response.status.must_equal 200
     end
   end
+
+  context 'when the buildkite_release_params hook gets trigger' do
+    before do
+      project.releases.destroy_all
+      project.builds.destroy_all
+      Integrations::BuildkiteController.any_instance.stubs(:deploy?).returns(true)
+      Integrations::BuildkiteController.any_instance.stubs(:project).returns(project)
+      Integrations::BuildkiteController.any_instance.stubs(:commit).returns(commit)
+      Integrations::BuildkiteController.any_instance.stubs(:branch).returns('master')
```

Comment:
Yep, they are not all necessary, I've reduce the number of stub to the minimum required in the latest commit